src/edu/csus/ecs/pc2/services/eventFeed/WebServer.java

```diff
@@ -365,6 +368,9 @@ private ResourceConfig getResourceConfig() {
             showMessage("Starting /contest/event-feed web service");
             resConfig.register(new StateService(getContest(), getController()));
             showMessage("Starting /contest/state web service");
+            resConfig.register(new VersionService(getContest(), getController()));
```

Comment:
The endpoint is supposed to be named the topmost level of the server since it is server specific, not contest specific.  See the example in the 2022-07 spec: https://example.com/api   and a typical endpoint for a contest at that server is: https://example.com/api/contests/wf14/teams

Reply:
pushed fix.  Changed version end point from /contests/version to /
ex.  https://localhost:50443/

Reply:
Then both the issue and PR should stop referring to a version endpoint.

Reply:
updated.